06-building-a-program-start-to-finish/writing-tests/output/spec/InvertedIndexSpec.js

```diff
@@ -0,0 +1,276 @@
+const newIndex = new InvertedIndex();
+
+describe('Tests for the InvertedIndex class', () => {
+
+  describe('Tests for the validateFile method', () => {
+    it('should return error message for non-Array objects', () => {
```

Comment:
Reading the reports actually felt out of place to me, but I ignored it I didn't know everything was supposed to read as a single sentence. Thanks a bunch for pointing it out.

I have implemented the pieces of feedback. Please review.